README.md

```diff
@@ -203,6 +204,16 @@ This command would generate the following configuration:
 }
 ```
 
+### Debugging rails applications
+
+Install `debug` gem. Verify by running `bundle exec rdbg -v`
+
+Make sure you're running your application with rdbg, e.g. `bundle exec rdbg -O -n -c -- bin/rails server -p 3000`
```

Comment:
```suggestion
Run your application with the debugger attached, so that the extension can connect to it.

bundle exec rdbg -O -n -c -- bin/rails server -p 3000
```